gulpfile.js

```diff
@@ -42,6 +43,12 @@ gulp.task('fonts', function() {
     .pipe(gulp.dest('./dist/fonts'));
 });
 
+gulp.task('webpack-token-worker', function() {
```

Comment:
I added it as a dependency to the 'webpack-app' task instead of 'serve'. I think it's better suited there since it's really part of the overall application.